holidays/countries/nepal.py

```
@@ -0,0 +1,441 @@
+        2018: (JUN, 16),
+        2019: (JUN, 5),
+        2020: (MAY, 25),
+        2021: (MAY, 14),
+        2022: (MAY, 3),
+        2023: (APR, 22),
+        2024: (APR, 11),
+        2025: (MAR, 31),
+    }
+
+
+class NepalHinduHolidays(_CustomHinduHolidays):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)